tests/testthat/test-weighting.R

```diff
@@ -124,5 +152,12 @@ with_test_authentication({
                 array(c(110, 100), dim=2L, dimnames=list(v4=c("B", "C"))))
         })
     })
-
+with(test.dataset(df), {
+        test_that("makeWeight returns the expected weights", {
+            ds$weight <- makeWeight(ds$v4 ~ c(30, 70, 0), name = "weight")
+            expected_weights <- c(0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4,
+                                  0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4)
```

Comment:
Oh, the thing which is divided by 100 are the targets, just so the user can type ``c(50, 50)` rather than `c(.5,.5)`. The expected weights here are what comes off the server after the raking has been done.